key_index.lua

```diff
@@ -0,0 +1,97 @@
+-- Storage to keep track of used keys. Allows to atomically create, delete
```

Comment:
I would prefer all source files in this repo to start with `prometheus` to make name collisions with other packages less likely. Can we either move this code to `prometheus.lua` or rename this file to `prometheus_keys.lua` (and add it to `.travis.yml` and `.rockspec`)?

Reply:
Done in 7d70eec0d90f22f6d07b8cd15f22d6b513a27ef1